BalanceSheet.py

```diff
@@ -191,9 +185,43 @@ def getGap2(percentAdd , percentSubb ):
 
 print("diff = +", diff, " %") # 9%
 
-
+#Idea:
 class partialEntry:
-    pass
+    """ captures a specific part of a `transaction` which can  either assume the  drSide or the crSide
+    Purpose: acts as a bridge between a Report (balanceSheet in this instance` and a transction (of a Journal Entry )
+
+    - Hint: only if we can add this as a form of function, in a transaction class, instead, that would have made things more "Straight-Forward, & easier
+            (For all users: developers & end-users alike)
+    """
+    #pass
+    def __init__(self):
+        pass
+    def __init__(self, drSide, crSide, amount, isDebit=True, MsgError="Unexpected Error Occured, please recheck input, then retry"):
+
+        # Check `isDebit` flag , if so, return the debitSide Object & the amount
+        if isDebit == True:
+            self.transactionSide = drSide
+            self.amount = amount
+            return self.transactionSide, self.amount
+        # otherwise, return `crSide` and the amount provided (in the consructor 
+        elif isDebit == False:
+            self.transactionSide = crSide
+            self.amount = amount
+            return self.transactionSide, self.amount
+
+        else:
+            raise ValueError(MsgError)
+
+# Now we can capture any part (of any transaction ) with its corresponding amount, accordingly
+# take the debit side (of a transaction) with its amount
+pEntry1 = partialEntry("cash","bankAccount", 100, isDebit=True)
+
+# take the credife side (of a transaction) with its amount
+pEntry2 = partialEntry("cash","bankAccount", 100, isDebit=False)

```

Comment:
`pEntry2` captures the Credit side of the transaction (i.e. `bankAccount` with amount `100` )